backend-implicaction/src/main/java/com/dynonuggets/refonteimplicaction/dto/forum/CategoryDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.dynonuggets.refonteimplicaction.dto.forum;
+
+import lombok.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Getter
+public class CategoryDTO {
```

Comment:
```suggestion
public class CategoryDto {
```
J'vais être un p'tit peu casse couille question naming dsl 😉 juste histoire de garder de la cohérence.

Reply:
D'accord ça va être fait 🙂